packages/valory/contracts/service_registry/tests/test_contract.py

```diff
@@ -136,3 +136,23 @@ def test_get_agent_instances(self) -> None:
             )
 
         assert result == return_value
+
+    def test_get_service_owner(self) -> None:
+        """Test service owner retrieval."""
+        service_owner = "0x5aAeb6053F3E94C9b9A09f33669435E7Ef1BeAed"
+
+        assert self.contract_address is not None
+
+        with mock.patch.object(
+            self.ledger_api,
+            "contract_method_call",
+            return_value=service_owner,
+        ):
+            actual = self.contract.get_service_owner(
+                self.ledger_api,
+                self.contract_address,
+                VALID_SERVICE_ID,
+            )
+
+        expected = dict(service_owner=service_owner)
+        assert expected == actual
```

Comment:
I followed the rest of the test in the contract, but I am not sure mocking like this is a good idea. It assumes that the the mocked return value has the same format as the contract. For this case it's obvious because it's an address only.